qiskit/circuit/random/utils.py

```diff
@@ -11,15 +11,335 @@
 # that they have been altered from the originals.
 
 """Utility functions for generating random circuits."""
-
 import numpy as np
 
-from qiskit.circuit import ClassicalRegister, QuantumCircuit, CircuitInstruction
-from qiskit.circuit import Reset
+from qiskit.circuit import (
+    ClassicalRegister,
+    QuantumCircuit,
+    CircuitInstruction,
+)
+from qiskit.circuit import Reset, Delay
 from qiskit.circuit.library import standard_gates
+from qiskit.circuit.library.standard_gates import get_standard_gate_name_mapping
 from qiskit.circuit.exceptions import CircuitError
 
 
+def _get_gates(n_qubits):
+    return [
```

Comment:
I have chosen not to return gates as np.ndarray, because it could happen that `_get_gates()` is used in some future functions where ndarray is not required?